ci/jenkins/test.sh

```diff
@@ -240,7 +240,9 @@ function collect_windows_network_info_and_logs {
 
         DOCKER_LOG_PATH="${DEBUG_LOG_PATH}/${NODENAME}/docker"
         mkdir "${DOCKER_LOG_PATH}"
-        scp -q -o StrictHostKeyChecking=no -T administrator@${IP}:'/cygdrive/c/"Program Files"/Docker/dockerd.log*' "${DOCKER_LOG_PATH}"
+        if [[ ${TESTCASE} != *"containerd"* ]];then
+            scp -q -o StrictHostKeyChecking=no -T administrator@${IP}:'/cygdrive/c/"Program Files"/Docker/dockerd.log*' "${DOCKER_LOG_PATH}"
```

Comment:
@XinShuYang do we need to collect any log for containerd?

Reply:
For containerd testbed we don't need dockerd.log